_data/ui/destination-settings-page.yml

```diff
@@ -99,7 +99,9 @@ postgres:
     copy: *password-copy
   - field: *database
     copy: *database-copy
-
+  - field: *ssl
+    copy: Connect with SSL. If a certificate is provided (typically a CA or server certificate) the connection will only succeed if the server's certificate verifies against it. 
```

Comment:
@cmerrick When you say `If a certificate is provided`, does that mean the cert the user can provide in the field that displays when the SSL box is checked, or something else?

Additionally, is there anything specific the user should know about providing Stitch with a cert in the UI?

Reply:
Yes, I mean the cert can be provided in the text box that is displayed

Some users will definitely have questions about which cert to provide or how to find/generate the cert - the docs for that could fill pages, so I'm reluctant to take it on ourselves.